app/views/shared/_banner.html.erb

```diff
@@ -50,6 +50,7 @@
             </div>
           </div>
         </div>
+        <button class="mobile close-banner-how-you-know" aria-label="close banner popup"></button>
```

Comment:
Do we want to translate the `aria-label`?

Reply:
Ugh...good call.  Going too fast.  Thx.